Content/Items/Gear/Gear.cs

```diff
@@ -297,6 +297,8 @@ public override void ModifyTooltips(List<TooltipLine> tooltips)
 
 		if (Item.damage > 0)
 		{
+			float damage1 = Main.LocalPlayer.GetDamage(DamageClass.Melee).ApplyTo(Item.damage);
+			float damage2 = Main.LocalPlayer.GetDamage(DamageClass.Generic).ApplyTo(Item.damage);

```

Comment:
Is this a fix to some damage issue?

Reply:
No, that was me testing if the damage affixes worked or not; deleted the write line but i missed them, lol.